wpilibc/athena/include/MultiSpeedController.h

```diff
@@ -0,0 +1,37 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016-2017. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <vector>
+
+#include "SpeedController.h"
+
+namespace frc {
+
+class MultiSpeedController : public SpeedController {
+ public:
+  MultiSpeedController();
```

Comment:
Since this is an empty constructor, just do `MultiSpeedController() = default;` and don't provide an implementation.